src/SFA.DAS.LoginService.Database/Manual/CS-1163-Update.sql

```diff
@@ -0,0 +1,52 @@
+﻿
+  /*  CS-1163
+  
+  Update the SupportUrl eg. in AT
+  "SupportUrl": "https://confirm.at-aas.apprenticeships.education.gov.uk/"  to
+  "SupportUrl": "https://at-aas.apprenticeships.education.gov.uk/home"
+  
+  Prod:
+  "SupportUrl": "https://confirm.my.apprenticeships.education.gov.uk/"  to  
+  "SupportUrl": "https://my.apprenticeships.education.gov.uk/Home"
+
+  */
+
+  declare @environment nvarchar(10) =
+	case
+		when DB_NAME() = 'das-at-alogin-db' then 'at'
+		when DB_NAME() = 'das-test-alogin-db' then 'test'
+		when DB_NAME() = 'das-test2-alogin-db' then 'test2'
+		when DB_NAME() = 'das-pp-alogin-db' then 'pp'
+		when DB_NAME() = 'das-prd-alogin-db' then ''
+	else null
+  end;
+
+  if @environment is null
+	throw 60000, 'Database should be one of das-at-alogin-db, das-test-alogin-db, das-test2-alogin-db, das-pp-alogin-db, das-prd-alogin-db', 1;
```

Comment:
I suspect this throw does not terminate the connection (I think a level of 20 has to be passed in to force this). Therefore the rest of the script (I think will run) and this may cause problems. I've not tried this explicitly and would love to know if what I read is correct! So I'm very happy to be proved wrong

There are options to to wrap this update in an open transaction and only commit if the values are what we expect.

Reply:
When the 60000 is thrown, the rest of the script will not run